### PR TITLE
added wg-policy-spotlight blog

### LINKED ISSUE #533: Spotlight: WG Policy
Markdown is ready.

### PR DESCRIPTION
**Pull request by arujjval**
closes #533

### CODE REVIEW

**Review by ArvindParekh — COMMENTED** [review]
How about moving the answers to the questions onto a new line to improve readability?

**Review by ArvindParekh — COMMENTED** [review]
Was going through this again and found some nits :)

**Comment by arujjval** [comment]
@ArvindParekh updated as per your comments.

**Review by bashlion — APPROVED** [review]

**Comment by chris-short** [comment]
Please update the date. After that, I'll lgtm and approve and we'll get this published.

**Comment by sftim** [comment]
/hold

We'd usually mirror this kind of article to the main blog. Are we happy to hold off to allow for that?

OK to unhold if we don't want mirroring.

**Comment by chris-short** [comment]
Yes, we should mirror (thank you, vacation brain is real).

**Comment by kaslin** [comment]
Do we need to do anything in particular for the mirroring? @arujjval we need to get a new publish date for this, right?

**Comment by sftim** [comment]
Needs a contributor to open a k/website PR that mirrors this one.
It's OK to put `draft: true` in front matter on these BTW.

**Comment by arujjval** [comment]
@sftim so I have to make a similar PR at [here](https://github.com/arujjval/website/tree/main/content/en/blog/_posts)?

Also, need to discuss the date it needs to get published.

**Comment by chris-short** [comment]
Correct, @arujjval, that's the right place. We would want to copy it there and set the canonical URL to the contributor-site URL. In that PR for the website repo, state that reviews are taking place in the contributor-site pull request and to request edits on this [PR](https://github.com/kubernetes/contributor-site/pull/534)

**Comment by kaslin** [comment]
Ping, any updates here @arujjval? Help needed?

**Comment by arujjval** [comment]
@kaslin been a little busy lately. 

Have we decided on which date this should be posted?

**Comment by sftim** [comment]
I suggest aiming for publication in late February, because SIG Docs is struggling with blog team capacity (it's mostly me).

**Comment by arujjval** [comment]
@sftim sure. Please update me about the day chosen. I will make changes to the publication date.

File: content/en/blog/2024/wg-policy-spotlight-2024.md [hunk 1]
```diff
@@ -0,0 +1,102 @@
+---
+layout: blog
+title: "Spotlight on Policy Working Group"
+slug: wg-policy-spotlight-2024
+date: 2024-09-25
```
Comment (reylejano) [inline content/en/blog/2024/wg-policy-spotlight-2024.md:5]: Please update the date, below is only a suggestion
```suggestion
date: 2025-02-26
```

Comment (sftim) [reply content/en/blog/2024/wg-policy-spotlight-2024.md:5]: How about adding `draft: true`. Then we can [aim to] merge it as a draft and later send in a small PR to get the article published.

**Comment by kaslin** [comment]
@arujjval checking in on this. Looks like there are some comments to address.

File: content/en/blog/2024/wg-policy-spotlight-2024.md (hunk 1 above)
Comment (arujjval) [reply content/en/blog/2024/wg-policy-spotlight-2024.md:5]: Should I put the date as 9th March now?

File: content/en/blog/2024/wg-policy-spotlight-2024.md [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+author: "Arujjwal Negi"
+---  
+
+In the complex world of Kubernetes, policies play a crucial role in managing and securing clusters. But have you ever wondered how these policies are developed, implemented, and standardized across the Kubernetes ecosystem? To answer that, let's put the spotlight on the Policy Working Group.
+
+The Policy Working Group is dedicated to a critical mission: providing an overall architecture that encompasses both current policy-related implementations and future policy proposals in Kubernetes. Their goal is ambitious yet essential - to create a universal view of policy architecture that serves both developers and end-users alike.
```
Comment (graz-dev) [inline content/en/blog/2024/wg-policy-spotlight-2024.md:11]: > Their goal is ambitious yet essential - to create a universal view of policy architecture that serves both developers and end-users alike.

Maybe this sentence could be polished as follow:

> Their goal is both ambitious and essential: to develop a universal policy architecture that benefits developers and end-users alike.

File: content/en/blog/2024/wg-policy-spotlight-2024.md [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+
+**Andy Suderman**: My name is Andy Suderman and I am the CTO of Fairwinds, a managed Kubernetes-as-a-Service provider. I began working with Kubernetes in 2016 building a web conferencing platform. I am an author and/or maintainer of several Kubernetes-related open-source projects such as Goldilocks, Pluto, and Polaris. Polaris is a JSON-schema-based policy engine, which started Fairwinds' journey into the policy space and my involvement in the Policy Working Group.
+
+**Poonam Lamba**: My name is Poonam Lamba, and I currently work as a Product Manager for Google Kubernetes Engine (GKE) at Google. My journey with Kubernetes began back in 2017 when I was building an SRE platform for a large enterprise, using a private cloud built on Kubernetes. Intrigued by its potential to revolutionize the way we deployed and managed applications at the time, I dove headfirst into learning everything I could about it.
+
```
Comment (graz-dev) [inline content/en/blog/2024/wg-policy-spotlight-2024.md:34]: Remove this space if the sentence below is the last part of the Poonam answer.

File: content/en/blog/2024/wg-policy-spotlight-2024.md [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+Since then, I've had the opportunity to build the policy and compliance products for GKE. I lead and contribute to GKE CIS benchmarks. I am involved with the Gatekeeper project as well as I have contributed to Policy-WG for over 2 years currently I serve as a co-chair for K8s policy WG.
+
+*Response to further questions is represented as an amalgamation of responses from co-chairs*
+
+## About Working Groups
+
+**One thing even I am not aware of is the difference between a working group and a SIG. Can you help us understand what a working group is and how it is different from a SIG?**
+Unlike SIGs, working groups are temporary and focused on tackling specific, cross-cutting issues or projects that may involve multiple SIGs. Their lifespan is defined, and they disband once they've achieved their objective. Generally, working groups don't own code or have long-term responsibility for managing a particular area of the Kubernetes project.
```
Comment (graz-dev) [inline content/en/blog/2024/wg-policy-spotlight-2024.md:42]: Adding an empty line between the questions and the answers could improve readibility. 
Then here you can add the link to the SiG's list (https://github.com/kubernetes/community/blob/master/sig-list.md) to let the readers aware of all the groups available.

File: content/en/blog/2024/wg-policy-spotlight-2024.md [hunk 5]
```diff
@@ -0,0 +1,102 @@
...
+ 
+**As you mentioned Working groups involve multiple SIGS, what SIGS are you closely involved with and How do you coordinate with them?**
```
Comment (graz-dev) [inline content/en/blog/2024/wg-policy-spotlight-2024.md:44]: Remove the capital letter at "How" if there is the same sentence.

File: content/en/blog/2024/wg-policy-spotlight-2024.md [hunk 6]
```diff
@@ -0,0 +1,102 @@
...
+The charter of the Policy WG is to help standardize policy management for Kubernetes and educate the community on best practices.
+
+To accomplish this we have updated the Kubernetes documentation ([Policies | Kubernetes](https://kubernetes.io/docs/concepts/policy)), produced several whitepapers ([Kubernetes Policy Management](https://github.com/kubernetes/sig-security/blob/main/sig-security-docs/papers/policy/CNCF_Kubernetes_Policy_Management_WhitePaper_v1.pdf), [Kubernetes GRC](https://github.com/kubernetes/sig-security/blob/main/sig-security-docs/papers/policy_grc/Kubernetes_Policy_WG_Paper_v1_101123.pdf)), and created the Policy Reports API ([API reference](https://htmlpreview.github.io/?https://github.com/kubernetes-sigs/wg-policy-prototypes/blob/master/policy-report/docs/index.html)) which standardizes reporting across different various tools. Several popular tools such as Falco, Trivy, Kyverno, kube-bench, and others support the Policy Report API. A major milestone for the Policy WG will be to help promote the Policy Reports API to a SIG-level API or find another stable home for it.
```
Comment (graz-dev) [inline content/en/blog/2024/wg-policy-spotlight-2024.md:66]: > ... which standardizes reporting across different various tools.

Keep just one between "different" and "various" becouse keeping boths could sounds like a redundance.

File: content/en/blog/2024/wg-policy-spotlight-2024.md [hunk 7]
```diff
@@ -0,0 +1,102 @@
...
+
+Beyond that, as ValidatingAdmissionPolicy and MutatingAdmissionPolicy become GA in Kubernetes, we intend to guide and educate the community on the tradeoffs and appropriate usage patterns for these built-in API objects and other CNCF policy management solutions like OPA/Gatekeeper and Kyverno.
```
Comment (graz-dev) [inline content/en/blog/2024/wg-policy-spotlight-2024.md:68]: You can add a link to Validating Admission Policy (https://kubernetes.io/docs/reference/access-authn-authz/validating-admission-policy/) and Mutating Admission Policy(https://kubernetes.io/docs/reference/access-authn-authz/mutating-admission-policy/) docs here

**Review by graz-dev — COMMENTED** [review]
@arujjval great piece!
I reviewed the article and left some comments about formatting and improving readibility.

**Comment by sftim** [comment]
@graz-dev for reported speech, we don't suggest rewordings unless there are obvious problems or we think the original speaker may have been misquoted.

File: content/en/blog/2024/wg-policy-spotlight-2024.md [hunk 8]
```diff
@@ -0,0 +1,102 @@
...
+
+## Challenges
+
+**What are some of the major challenges that the Policy Working Group is working on or has worked on?**
+Throughout our work in the Policy Working Group, we've encountered several challenges.
```
Comment (jberkus) [inline content/en/blog/2024/wg-policy-spotlight-2024.md:73]:
```suggestion
During our work in the Policy Working Group, we've encountered several challenges:
```

File: content/en/blog/2024/wg-policy-spotlight-2024.md [hunk 9]
```diff
@@ -0,0 +1,102 @@
...
+
+Additionally, differences of opinion are valued and encouraged within the Policy-WG. We adhere to the CNCF core values and resolve disagreements by maintaining respect for one another. We also strive to timebox our decisions and assign clear responsibilities to keep things moving forward.
+
+
+## New Contributors
+
+**What skills are expected from new contributors and how can they get involved with the Policy Working Group?**
+The Policy WG is ideal for anyone who is passionate about Kubernetes security, governance, and compliance and wants to help shape the future of how we build, deploy, and operate cloud-native workloads.
+
+Join the mailing list as described on our community [page](https://github.com/kubernetes/community/blob/master/wg-policy/README.md) and attend one of our upcoming community meetings.
```
Comment (jberkus) [inline content/en/blog/2024/wg-policy-spotlight-2024.md:96]:
```suggestion
Join the mailing list as described on our community [page](https://github.com/kubernetes/community/blob/master/wg-policy/README.md) and attend one of our upcoming [community meetings](https://github.com/kubernetes/community/tree/master/wg-policy#meetings).
```

**Review by jberkus — CHANGES_REQUESTED** [review]
Just some minor cleanup.  Also, please edit per @graz-dev review comments as well.

**Comment by jberkus** [comment]
@graz-dev good work on copy-edit review!

File: content/en/blog/2024/wg-policy-spotlight-2024.md (hunk 1 above)
Comment (lmktfy) [reply content/en/blog/2024/wg-policy-spotlight-2024.md:5]: Try this:
```suggestion
date: 2025-02-02
draft: true
```

I promise you it's the right change to make given our [guidance](https://kubernetes.io/docs/contribute/blog/) on blog contributions.

**Comment by arujjval** [comment]
Sorry for the delay. I have made changes as per the suggestions above.

File: content/en/blog/2024/wg-policy-spotlight-2024.md
Comment (lmktfy) [inline content/en/blog/2024/wg-policy-spotlight-2024.md:1]: Hi
Can we merge this as a draft and then open small PRs to actually publish it?

If that sounds OK, please set they year (see filename) to 2025.

Comment (arujjval) [reply content/en/blog/2024/wg-policy-spotlight-2024.md:1]: Sure. I have changed the title per your request.

File: content/en/blog/2025/wg-policy-spotlight-2025.md [hunk 10]
```diff
@@ -0,0 +1,110 @@
+---
+layout: blog
+title: "Spotlight on Policy Working Group"
+slug: wg-policy-spotlight-2025
+date: 2025-02-02
```
Comment (lmktfy) [inline content/en/blog/2025/wg-policy-spotlight-2025.md:5]:
```suggestion
date: 2025-05-22
draft: true # remove this line to mark for publication
```

Comment (lmktfy) [reply content/en/blog/2025/wg-policy-spotlight-2025.md:5]: Please make a similar change to the k/website PR.

**Comment by kaslin** [comment]
@arujjval You have a publishing date! Check out the new edits and approve them so this can go out on time.

**Comment by arujjval** [comment]
@kaslin @sftim I have made changes and updated the website PR as per suggestions.

File: content/en/blog/2025/wg-policy-spotlight-2025.md [hunk 11]
```diff
@@ -0,0 +1,110 @@
+---
+layout: blog
+title: "Spotlight on Policy Working Group"
+slug: wg-policy-spotlight-2025
+date: 2025-05-22
```
Comment (lmktfy) [inline content/en/blog/2025/wg-policy-spotlight-2025.md:5]:
```suggestion
draft: true
date: 2025-05-22
```

We'll merge this as a draft and then send in follow-up PRs that remove the `draft: true` part.

File: content/en/blog/2025/wg-policy-spotlight-2025.md [hunk 12]
```diff
@@ -0,0 +1,110 @@
...
+author: "Arujjwal Negi"
+---  
+
+In the complex world of Kubernetes, policies play a crucial role in managing and securing clusters. But have you ever wondered how these policies are developed, implemented, and standardized across the Kubernetes ecosystem? To answer that, let's put the spotlight on the Policy Working Group.
+
+The Policy Working Group is dedicated to a critical mission: providing an overall architecture that encompasses both current policy-related implementations and future policy proposals in Kubernetes. Their goal is both ambitious and essential: to develop a universal policy architecture that benefits developers and end-users alike.
+
+Through collaborative methods, this working group is striving to bring clarity and consistency to the often complex world of Kubernetes policies. By focusing on both existing implementations and future proposals, they're working to ensure that the policy landscape in Kubernetes remains coherent and accessible as the technology evolves.
+
+In this blog post, we'll dive deeper into the work of the Policy Working Group, guided by insights from its co-chairs:
```
Comment (lmktfy) [inline content/en/blog/2025/wg-policy-spotlight-2025.md:15]:
```suggestion
In this blog post, I'll dive deeper into the work of the Policy Working Group, guided by insights from its co-chairs:
```

per https://kubernetes.io/docs/contribute/blog/article-submission/#article-content (we try to avoid “we“, especially when it's unclear which group is meant)

File: content/en/blog/2025/wg-policy-spotlight-2025.md [hunk 13]
```diff
@@ -0,0 +1,110 @@
...
+**What skills are expected from new contributors and how can they get involved with the Policy Working Group?**
+
+The Policy WG is ideal for anyone who is passionate about Kubernetes security, governance, and compliance and wants to help shape the future of how we build, deploy, and operate cloud-native workloads.
+
+Join the mailing list as described on our community [page](https://github.com/kubernetes/community/blob/master/wg-policy/README.md) and attend one of our upcoming [community meetings](https://github.com/kubernetes/community/tree/master/wg-policy#meetings).
+
+
+---
+
+This is where our discussion about the Policy Working Group ends. We hope this gave you some insights into the group's aims and workings. Of course, this is just the tip of the iceberg. To learn more and get involved with the Policy Working Group, consider attending their meetings. You can find the schedule and join their discussions [here](https://github.com/kubernetes/community/tree/master/wg-policy).
```
Comment (lmktfy) [inline content/en/blog/2025/wg-policy-spotlight-2025.md:109]: Maybe:
```suggestion
This is where our discussion about the Policy Working Group ends. The working group, and especially the people who took part in this article, hope this gave you some insights into the group's aims and workings. Of course, this is just the tip of the iceberg. To learn more and get involved with the Policy Working Group, consider attending their meetings. You can find the schedule and join their [discussions](https://github.com/kubernetes/community/tree/master/wg-policy).
```

per https://kubernetes.io/docs/contribute/blog/article-submission/#article-content (we try to avoid “we“, especially when it's unclear which group is meant); also, we avoid "here".

File: content/en/blog/2025/wg-policy-spotlight-2025.md [hunk 14]
```diff
@@ -0,0 +1,110 @@
...
+**Can you tell me more about those challenges? How did you discover each one? What has the impact been? Do you have ideas or strategies about how to address them?**
+
+There are no easy answers, but having more contributors and maintainers greatly helps! Overall the CNCF community is great to work with and is very welcoming to beginners. So, if folks out there are hesitating to get involved, I highly encourage them to attend a WG or SIG meeting and just listen in.
+
+It often takes a few meetings to fully understand the discussions, so don't feel discouraged if you don't grasp everything right away. We've started emphasizing this point and encourage new members to review documentation as a starting point for getting involved.
+
+Additionally, differences of opinion are valued and encouraged within the Policy-WG. We adhere to the CNCF core values and resolve disagreements by maintaining respect for one another. We also strive to timebox our decisions and assign clear responsibilities to keep things moving forward.
+
+
+## New Contributors
```
Comment (lmktfy) [inline content/en/blog/2025/wg-policy-spotlight-2025.md:98]: (nit)
```suggestion
## New contributors
```

File: content/en/blog/2025/wg-policy-spotlight-2025.md [hunk 15]
```diff
@@ -0,0 +1,110 @@
...
+
+Through collaborative methods, this working group is striving to bring clarity and consistency to the often complex world of Kubernetes policies. By focusing on both existing implementations and future proposals, they're working to ensure that the policy landscape in Kubernetes remains coherent and accessible as the technology evolves.
+
+In this blog post, we'll dive deeper into the work of the Policy Working Group, guided by insights from its co-chairs:
+
+-   [Jim Bugwadia](https://twitter.com/JimBugwadia)
+-   [Poonam Lamba](https://twitter.com/poonam-lamba)
+-   [Andy Suderman](https://twitter.com/sudermanjr)
+
+Interviewed by [Arujjwal Negi](https://twitter.com/arujjval).
```
Comment (lmktfy) [inline content/en/blog/2025/wg-policy-spotlight-2025.md:21]: _Interviewed by [Arujjwal Negi](https://twitter.com/arujjval)._

File: content/en/blog/2025/wg-policy-spotlight-2025.md [hunk 16]
```diff
@@ -0,0 +1,110 @@
...
+
+These co-chairs will let us know what the Policy working group is all about.
```
Comment (lmktfy) [inline content/en/blog/2025/wg-policy-spotlight-2025.md:23]:
```suggestion
These co-chairs will explain what the Policy working group is all about.
```
per https://kubernetes.io/docs/contribute/blog/article-submission/#article-content (we try to avoid “we“ and “us”, especially when it's unclear which group is meant)

File: content/en/blog/2025/wg-policy-spotlight-2025.md [hunk 17]
```diff
@@ -0,0 +1,110 @@
...
+
+*Response to further questions is represented as an amalgamation of responses from co-chairs*
+
+## About Working Groups
+
+**One thing even I am not aware of is the difference between a working group and a SIG. Can you help us understand what a working group is and how it is different from a SIG?**
+
+Unlike SIGs, working groups are temporary and focused on tackling specific, cross-cutting issues or projects that may involve multiple SIGs. Their lifespan is defined, and they disband once they've achieved their objective. Generally, working groups don't own code or have long-term responsibility for managing a particular area of the Kubernetes project.
+
+(To know more about SIGs, visit [here](https://github.com/kubernetes/community/blob/master/sig-list.md))
```
Comment (lmktfy) [inline content/en/blog/2025/wg-policy-spotlight-2025.md:43]: (nit)
```suggestion
(To know more about SIGs, visit the [list of Special Interest Groups](https://github.com/kubernetes/community/blob/master/sig-list.md))
```

File: content/en/blog/2025/wg-policy-spotlight-2025.md (hunk 12 above)
Comment (lmktfy) [reply content/en/blog/2025/wg-policy-spotlight-2025.md:15]: I forgot to say before, but it's important.

During the spoken part of the discussion, “we” has whatever meaning the people speaking intended it to have, and that's fine.

File: content/en/blog/2025/wg-policy-spotlight-2025.md (hunk 16 above)
Comment (lmktfy) [reply content/en/blog/2025/wg-policy-spotlight-2025.md:23]: I forgot to say before, but it's important.

During the spoken part of the discussion, “we” has whatever meaning the people speaking intended it to have, and that's fine.

File: content/en/blog/2025/wg-policy-spotlight-2025.md (hunk 13 above)
Comment (lmktfy) [reply content/en/blog/2025/wg-policy-spotlight-2025.md:109]: I forgot to say before, but it's important.

During the spoken part of the discussion, “we” has whatever meaning the people speaking intended it to have, and that's fine.

**Review by lmktfy — COMMENTED** [review]
The key thing I hope we get right is to mark this as a draft, because that fits with how the blog team (for the main blog) plan to work.

See https://github.com/kubernetes/contributor-site/pull/534/files#r2094645018

However, I also suggested a few little tweaks that I think are worth making.

**Comment by lmktfy** [comment]
Looks good to merge as draft.

Thanks for the work that went into this. I'll also get it scheduled & published.

/LGTM
/approve

**Review by jberkus — APPROVED** [review]

**Comment by arujjval** [comment]
@jberkus I guess the content needs to be the same, just need to do some editing on the grammar?

**Comment by kaslin** [comment]
@jberkus & @arujjval reminder to work out how to modify this blog since the WG disbanded.

**Comment by arujjval** [comment]
> This is painful to do, but this needs to go back to content before we publish; [WG-Policy has concluded its work and disbanded](https://github.com/kubernetes/community/pull/8489), and as such this blog post needs to be edited to reflect that new status. The blog needs to turn into a retro of "this is what Policy accomplished".

I have made changes according to this. So, the blog is up for review again.

**Comment by lmktfy** [comment]
/lgtm
/approve

We can merge this, as an unpublished draft. Next step is to copy that updated text into a PR against [k/website](https://github.com/Kubernetes/website), and once that has merged we can set a publication date. Soon is good.

**Comment by arujjval** [comment]
@lmktfy done! We can now discuss the publishing date.

**Comment by kaslin** [comment]
@lmktfy Looks like this might be ready. Can we get it on the schedule?

File: content/en/blog/2025/wg-policy-spotlight-2025.md [hunk 18]
```diff
@@ -0,0 +1,103 @@
...
+
+
+**Can you tell us what were the main objectives of the Policy Working Group and some of your key accomplishments?**
+
+The charter of the Policy WG was to help standardize policy management for Kubernetes and educate the community on best practices.
+
+To accomplish this we updated the Kubernetes documentation ([Policies | Kubernetes](https://kubernetes.io/docs/concepts/policy)), produced several whitepapers ([Kubernetes Policy Management](https://github.com/kubernetes/sig-security/blob/main/sig-security-docs/papers/policy/CNCF_Kubernetes_Policy_Management_WhitePaper_v1.pdf), [Kubernetes GRC](https://github.com/kubernetes/sig-security/blob/main/sig-security-docs/papers/policy_grc/Kubernetes_Policy_WG_Paper_v1_101123.pdf)), and created the Policy Reports API ([API reference](https://htmlpreview.github.io/?https://github.com/kubernetes-sigs/wg-policy-prototypes/blob/master/policy-report/docs/index.html)) which standardizes reporting across various tools. Several popular tools such as Falco, Trivy, Kyverno, kube-bench, and others support the Policy Report API. A major milestone for the Policy WG was promoting the Policy Reports API to a SIG-level API or finding it a stable home.
```
Comment (pacoxu) [inline content/en/blog/2025/wg-policy-spotlight-2025.md:74]: The link here needs an update. 
- After https://github.com/kubernetes-retired/wg-policy-prototypes/pull/135, the api reference was moved to https://github.com/kubernetes-retired/wg-policy-prototypes/blob/master/policy-report/docs/api-docs.md. 

```suggestion
To accomplish this we updated the Kubernetes documentation ([Policies | Kubernetes](https://kubernetes.io/docs/concepts/policy)), produced several whitepapers ([Kubernetes Policy Management](https://github.com/kubernetes/sig-security/blob/main/sig-security-docs/papers/policy/CNCF_Kubernetes_Policy_Management_WhitePaper_v1.pdf), [Kubernetes GRC](https://github.com/kubernetes/sig-security/blob/main/sig-security-docs/papers/policy_grc/Kubernetes_Policy_WG_Paper_v1_101123.pdf)), and created the Policy Reports API ([API reference](https://github.com/kubernetes-retired/wg-policy-prototypes/blob/master/policy-report/docs/api-docs.md)) which standardizes reporting across various tools. Several popular tools such as Falco, Trivy, Kyverno, kube-bench, and others support the Policy Report API. A major milestone for the Policy WG was promoting the Policy Reports API to a SIG-level API or finding it a stable home.
```

**Comment by arujjval** [comment]
I have updated it as per @pacoxu suggestions
